src/screens/SendCoinsScreen.tsx

```diff
@@ -377,7 +377,15 @@ class SendCoinsScreen extends Component<Props, State> {
                 }),
               );
             } catch (error) {
-              Alert.alert(error.message);
+              CreateMessage({
```

Comment:
on going back you would reset all the form

Reply:
Why? If someone scanned the wrong QRcode it is better to have the send form filled?

Reply:
No, it behaves like a typical onBack call.

Reply:
we can go screen back (isn't this happening now anyway?) but the form should be filled as it was before scanning

Reply:
oh ok, so the form is still filled